hack/ci/create-cluster.sh

```diff
@@ -3,40 +3,57 @@ set -o errexit
 
 REPO_ROOT="${REPO_ROOT:-$(dirname "${BASH_SOURCE}")/../..}"
 KUBECTL_BIN="${KUBECTL_BIN:-$REPO_ROOT/bin/kubectl}"
+CMCTL_BIN="${CMCTL_BIN:-$REPO_ROOT/bin/cmctl}"
 HELM_BIN="${HELM_BIN:-$REPO_ROOT/bin/helm}"
 KIND_BIN="${KIND_BIN:-$REPO_ROOT/bin/kind}"
-POLICY_APPROVER_TAG="${POLICY_APPROVER_TAG:-smoke}"
-POLICY_APPROVER_REPO="${POLICY_APPROVER_REPO:-quay.io/jetstack/cert-manager-approver-policy}"
-POLICY_APPROVER_IMAGE="$POLICY_APPROVER_REPO:$POLICY_APPROVER_TAG"
+CSI_TAG="${CSI_TAG:-smoke}"
+CSI_DRIVER_REPO="${CSI_DRIVER_REPO:-quay.io/jetstack/cert-manager-csi-driver-spiffe}"
+CSI_DRIVER_IMAGE="$CSI_DRIVER_REPO:$CSI_TAG"
+CSI_APPROVER_REPO="${CSI_APPROVER_REPO:-quay.io/jetstack/cert-manager-csi-driver-spiffe-approver}"
+CSI_APPROVER_IMAGE="$CSI_APPROVER_REPO:$CSI_TAG"
 
-echo ">> building approver-policy binary..."
-GOARCH=$(go env GOARCH) GOOS=linux CGO_ENABLED=0 go build -o $REPO_ROOT/bin/approver-policy-linux $REPO_ROOT/cmd/.
+echo ">> building docker images..."
+docker build -t $CSI_DRIVER_IMAGE -f Dockerfile.driver .
+docker build -t $CSI_APPROVER_IMAGE -f Dockerfile.approver .
```

Comment:
Hmm, no issues from me (or CI). I'm running v20.10.9. Not sure what version CI is using.

Reply:
CI uses an ancient version I think https://github.com/jetstack/testing/blob/master/images/bazelbuild/Dockerfile#L50.

But I don't want to hold this PR because of this 👍🏼